### PR TITLE
tests: port first regression test from snap-confine

### PR DESCRIPTION
**Pull request by zyga**

### CODE REVIEW

File: spread.yaml [hunk 1]
```diff
@@ -259,6 +259,24 @@ suites:
         restore-each: |
             $TESTSLIB/reset.sh --reuse-core
 
+    tests/regression/:
+        summary: Regression tests for snapd
+        systems: [-ubuntu-core-16-64-fixme]
+        prepare: |
+            . $TESTSLIB/prepare.sh
+            if [[ "$SPREAD_SYSTEM" == ubuntu-core-16-* ]]; then
+                prepare_all_snap
```
Comment (fgimenez) [inline spread.yaml]: This won't work on the boards, the systems are `ubuntu-core-16-64`, `ubuntu-core-16-arm-32` and `ubuntu-core-16-arm-64`, maybe better to use the `external` backend as a filter.

File: tests/lib/snaps/shell/meta/snap.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name: shell
+version: 1.0
+summary: Simple POSIX shell snap
+description: |
+    This snap contains a simple shell.
+    It is a replacement for snapd-hacker-toolbelt.
+apps:
+    shell:
+        command: ../../../bin/sh
```
Comment (fgimenez) [inline tests/lib/snaps/shell/meta/snap.yaml]: Not totally sure but maybe you could use the `test-snapd-tools.cmd` command, take a look at https://github.com/snapcore/snapd/blob/master/tests/lib/snaps/test-snapd-tools/bin/cmd

File: spread.yaml (hunk 1 above)
Comment (zyga) [inline spread.yaml]: I didn't know how to block the external backend, I listed all core systems mentioned in spread.yaml. Is that sufficient?

File: tests/lib/snaps/shell/meta/snap.yaml (hunk 2 above)
Comment (zyga) [inline tests/lib/snaps/shell/meta/snap.yaml]: You are right, I didn't need a new snap. Rewrite entirely to use `test-snapd-tools.cmd`

File: spread.yaml (hunk 1 above)
Comment (fgimenez) [inline spread.yaml]: Yes, I think that should be enough, we are doing the same for the main suite.

**Comment by stolowski** [comment]
LGTM! +1

**Pull request merged**